src/util/graph.h

```diff
@@ -601,6 +601,17 @@ void grapht<N>::tarjan(tarjant &t, node_indext v) const
   }
 }
 
+/// Populate a vector giving an SCC index for each node. For example, if nodes
```

Comment:
May I suggest starting with "Computes strongly-connected components of a graph and yields a map from nodes to components indices." I think this requires less context.